exercisePPJ_06_I/FunctionPPJ_06_I.java

```diff
@@ -0,0 +1,52 @@
+public class FunctionPPJ_06_I {
+	
+	private double binaryWrt[] = {0,0,0,0,0,0,0,0};
```

Comment:
It's already defined in ExercisePPJ_06_I.java
It's not a good idea to have two definitions of thing that should have the same value.

Reply:
@MatDls 
it's true, I will delete redundant variable